src/components/App.js

```diff
@@ -1,17 +1,35 @@
-import React from 'react';
+import React, { Component } from 'react';
 
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 // eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] We can safely remove this `eslint` disable comment here, it will not make a difference in your app.

Reply:
@Torres-ssf Thank you very much. 👍